### PR TITLE
Faraz/lp 1199

### PR DESCRIPTION
**Pull request by ahmed-faraz46**

### CODE REVIEW

**Comment by ahmed-faraz46** [comment]
@emraan004 @saad-waqar please review I'm merging to test facebook sharing can't be tested on local

File: openedx/features/student_certificates/tasks.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+@task(routing_key=settings.HIGH_MEM_QUEUE)
+def task_create_certificate_img_and_upload_to_s3(certificate):
+    """
+    :param certificate:
+    :return:
+    """
+    try:
+        url = get_certificate_url(certificate)
+        img_name = get_certificate_image_name(certificate)
+        uuid = certificate.verify_uuid
```
Comment (emraan004) [inline openedx/features/student_certificates/tasks.py]: let's move this above and pass this id in helper methods and modify them as well.

File: openedx/features/student_certificates/helpers.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+
+def upload_to_s3(file_path, s3_bucket, key_name):
+    """
+    :param file_path: path of the file we have to upload on s3
+    :param s3_bucket: bucket in which we have to upload
+    :param key_name: key by which we will place this file in the bucket
+    :return:
+    """
+    aws_access_key_id = getattr(settings, 'AWS_ACCESS_KEY_ID', None)
```
Comment (emraan004) [inline openedx/features/student_certificates/helpers.py]: let's create a helper method for connection and pass it to upload_to_s3

File: openedx/features/student_certificates/helpers.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    :return: return S3 key name for the image name
+    """
+    return '{prefix}/{img_name}'.format(prefix=CERTIFICATE_IMG_PREFIX, img_name=img_name)
+
+
+def get_philu_certificate_social_context(course, certificate):
+    custom_settings = get_course_custom_settings(certificate.course_id)
+    meta_tags = custom_settings.get_course_meta_tags()
+
+    tweet_text = meta_tags['title'] or TWITTER_TWEET_TEXT_FMT.format(
```
Comment (emraan004) [inline openedx/features/student_certificates/helpers.py]: let's remove meta title according to requirement

File: lms/djangoapps/philu_overrides/views.py [hunk 4]
```diff
@@ -628,6 +628,7 @@ def course_about(request, course_id):
 
         if course_details.banner_image_name != DEFAULT_IMAGE_NAME:
             meta_tags['image'] = settings.LMS_ROOT_URL + course_details.banner_image_asset_path
+        # meta_tags['image'] = "https://cdn1.imggmi.com/uploads/2019/3/12/d027cec46228c2519a3c4ac18793475f-full.jpg"
```
Comment (emraan004) [inline lms/djangoapps/philu_overrides/views.py]: let's remove it

File: openedx/features/student_certificates/constants.py [hunk 5]
```diff
@@ -0,0 +1,6 @@
+TWITTER_TWEET_TEXT_FMT = 'I just completed @PhilanthropyUni\'s free online {course_name} ' \
+                         'course and earned this certificate. Start learning today: ' \
+                         '{base_url}/{course_url}/{course_id}/{about_url}'
+TWITTER_META_TITLE_FMT = "I just completed Philanthropy University\'s {course_name} course!"
+SOCIAL_MEDIA_SHARE_URL_FMT = "{base_url}/achievements/{certificate_uuid}"
+COURSE_URL_FMT = "{base_url}/{course_url}/{course_id}/{about_url}"
```
Comment (emraan004) [inline openedx/features/student_certificates/constants.py]: please add endline here

**Pull request merged**